OpenRA.Mods.Common/Activities/ExternalCaptureActor.cs

```diff
@@ -49,8 +50,11 @@ public override Activity Tick(Actor self)
 			if ((nearest - mobile.ToCell).LengthSquared > 2)
 				return ActivityUtils.SequenceActivities(new MoveAdjacentTo(self, target), this);
 
-			if (!capturable.CaptureInProgress)
+			if (!capturable.CaptureInProgress) {
```

Comment:
Please note that the openra [coding standard](https://github.com/OpenRA/OpenRA/wiki/Coding-Standard) has braces on their own lines.  This applies to a few places in this PR.